nservicebus/msmq/routing.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Physical routing with MSMQ
```

Comment:
I think there is value in keeping them separate. It is perfectly valid to use the instance mapping file without even thinking about scaling out.